scripts/valgrind_suppressions_file.supp

```diff
@@ -5,4 +5,15 @@
    obj:/usr/*libmpich*
    fun:PMPI_Init
    fun:main
```

Comment:
```suggestion
```

Reply:
if you also delete the mpi_suppression_fortran it should be enough, if I understand the suppression correctly. 
Both errors come from the function PMPI_Init, but once it is called by C-MPI and once by fortran-MPI. The above suppression is probably to specific to catch it. 

If it doesn't work I am fine with your changes, too.